tox.ini

```diff
@@ -2,12 +2,12 @@
 envlist =
     lint,
     py35,py36,py37,py38,pypy3,
-    docs,py36-cover,coverage,
+    docs,coverage,
 
 [testenv]
 commands =
-    cover: coverage run \
-    {envbindir}/nosetests --with-xunit --xunit-file=nosetests-{envname}.xml {posargs:}
+    python --version
+    pytest {posargs:}
 extras =
     testing
 deps =
```

Comment:
the cover qualifier under here is not necessary since you added pytest-cov to the testing extra